orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteStageHandler.kt

```diff
@@ -54,7 +54,7 @@ class CompleteStageHandler(
       if (stage.status in setOf(RUNNING, NOT_STARTED)) {
         var status = stage.determineStatus()
         try {
-          if (status == NOT_STARTED || (status.isComplete && !status.isHalt)) {
+          if (status in setOf(RUNNING, NOT_STARTED) || (status.isComplete && !status.isHalt)) {
```

Comment:
I don't understand why this is changing, help.

Reply:
Nevermind.

Reply:
`determineStatus` didn't return `RUNNING` before.